artichoke-backend/src/extn/stdlib/mod.rs

```diff
@@ -22,6 +23,12 @@ pub fn init(interp: &Artichoke) -> Result<(), ArtichokeError> {
     Ok(())
 }
 
+pub mod base64 {
+    //! Ruby Base64 package, implemented with embedded sources from MRI 2.6.3.
+    // See scripts/auto_import/.
```

Comment:
oops this is a typo from the URI module below. `//!` indicates a doc comment applies to the whole module. Both comment lines here should start with `//!`.

@kenaniah would you be able to fix both the `base64` and `uri` module doc comments?